kedro-exercises/spaceflight/setup.cfg

```diff
@@ -1,3 +1,15 @@
+[flake8]
+max-line-length=88
+extend-ignore=E203
+
+[isort]
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+use_parentheses=True
+line_length=88
+known_third_party=kedro
+
 [tool:pytest]
 addopts=--cov-report term-missing
         --cov src/kedro_tutorial -ra
```

Comment:
`isort` and `pytest` are redundant here. They are configured in `toml` file

Reply:
Fixed.